spec/controllers/wishlists_controller_spec.rb

```diff
@@ -144,6 +216,34 @@
         include_examples "redirects to the 'sign_in' page"
       end
     end
+
+    describe "GET #edit" do
+      context "with valid parameters" do
+        before(:each) { get_edit(id: wishlist_id) }
+
+        include_examples "redirects to the 'sign_in' page"
+      end
+
+      context "with invalid parameters" do
+        before(:each) { post_create(id: -1) }
+
+        include_examples "redirects to the 'sign_in' page"
+      end
```

Comment:
I kept check of invalid params when user is logged out to show that the action is the same as when params are valid (that sign_up page is displayed, not sth like error 404). But maybe it's unnecessary indeed. I'll try to change schema of tests as You said.